mods/cnc/rules/defaults.yaml

```diff
@@ -668,8 +668,8 @@
 	ActorPreviewPlaceBuildingPreview:
 		OverridePalette: placebuilding
 	SoundOnDamageTransition:
-		DamagedSounds: xplos.aud
-		DestroyedSounds: crumble.aud
+		DamagedSounds: xplobig4.aud
+		DestroyedSounds: crumble.aud,xplobig4.aud
```

Comment:
style nit:
```suggestion
		DestroyedSounds: crumble.aud, xplobig4.aud
```

Reply:
https://github.com/OpenRA/OpenRA/pull/18967/files showed me we aren't enforcing a space after the comma so feel free to ignore

Reply:
It would be good to fix these, otherwise it makes more noise when we next run the utility to update rule changes.